static/_redirects

```diff
@@ -46,16 +46,16 @@
 /guides/traefik-ingress.html https://0-20-0.docs.pomerium.com/docs/guides/traefik-ingress 301!
 /docs/reference/forward-auth https://0-20-0.docs.pomerium.com/docs/reference/forward-auth 301!
 https://0-20-0.docs.pomerium.com/category/guides https://0-20-0.docs.pomerium.com/docs/guides
-/docs/reference/policy/allowed-domains https://0-14-0.docs.pomerium.com/reference
-/docs/reference/policy/allowed-idp-claims https://0-14-0.docs.pomerium.com/reference
-/docs/reference/policy/allowed-users https://0-14-0.docs.pomerium.com/reference
+/docs/reference/policy/allowed-domains https://0-14-0.docs.pomerium.com/reference/#allowed-domains
+/docs/reference/policy/allowed-idp-claims https://0-14-0.docs.pomerium.com/reference/#allowed-idp-claims
+/docs/reference/policy/allowed-groups https://0-14-0.docs.pomerium.com/reference/#allowed-groups
+/docs/reference/policy/allowed-users https://0-14-0.docs.pomerium.com/reference/#allowed-users
 /docs/reference/policy/policy-explanation https://0-14-0.docs.pomerium.com/reference
 /docs/reference/policy/policy-remediation https://0-14-0.docs.pomerium.com/reference
-/docs/reference/policy/policy https://0-14-0.docs.pomerium.com/reference
-
-# Link from v23 to v22
-/docs/reference/routes/headers#set-authorization-header https://0-22-0.docs.pomerium.com/reference/routes/headers#set-authorization-header
```

Comment:
I think this redirect will never trigger, because of the `#set-authorization-header` fragment on the source URL. (The fragment isn't part of the request sent to the server, so we can't issue a redirect based on it.) I figured we should just remove this line.